docs/tutorials/acme/http-validation.rst

```diff
@@ -33,7 +33,10 @@ You can read more about the Issuer resource in the :doc:`Issuer reference docs <
        privateKeySecretRef:
          name: letsencrypt-staging
        # Enable the HTTP-01 challenge provider
-       http01: {}
+       solvers:
+       - http01
+          ingress:
+            class: nginx
```

Comment:
🙈 looks like there's a missing `:` here...

```
       solvers:
       # An empty 'selector' means that this solver matches all domains
       - selector: {}
         http01:
           ingress:
             class: nginx
```

Reply:
sorry for my mistakes. let me fix it tmr.

thx for ur review

Reply:
done !